CONTRIBUTING.md

```diff
@@ -0,0 +1,178 @@
+# Contributing to the pyro-risks repository
+
+Everything you need to know to contribute efficiently to the project.
+
+
+## Codebase structure
+
+- [pyro-risks](https://github.com/pyronear/pyro-risks/tree/master/pyro_risks) - the pyro-risks library
+- [examples](https://github.com/pyronear/pyro-risks/tree/master/scripts) - examples scripts
+- [test](https://github.com/pyronear/pyro-vision/blob/master/test) - python unit tests
+
+
+## Continuous Integration
+
+This project uses the following integrations to ensure proper codebase maintenance:
+
+- [Github Actions](https://docs.github.com/en/free-pro-team@latest/actions/guides/about-continuous-integration) - run workflows for building and testing the package
+- [Codacy](https://www.codacy.com/) - analyzes commits for code quality
+- [Codecov](https://codecov.io/) - reports back coverage results
+
+As a contributor, you will only have to ensure coverage of your code by adding appropriate unit testing of your code.
+
+## Style conventions
+
+- **Code**:
+  - Setup the `__all__` special variable for each module
+  - Use type hints for every functions ([type hints cheat sheet](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html))
+  - Format your code using the [black](https://github.com/psf/black) auto-formatter
+  - Ensure to document your code using type hints compatible docstrings. In doing so, please follow [Google-style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) so it can ease the process of documentation later.
+
+- **Commit message**: please follow [Udacity guide](http://udacity.github.io/git-styleguide/)
+
+## Issues
+
+Use Github [issues](https://github.com/pyronear/pyro-risks/issues) for feature requests, or bug reporting. When doing so, use issue templates whenever possible and provide enough information for other contributors to jump in.
+
+## Code contribution
+
+In order to contribute to project, you will first need to **set up your *pyro-risks* development environment** and then follow the **contributing workflow** and the **code & commit guidelines**.
+
+* [Project Setup](#project-setup): *fork the project and install dependencies in a well-encapsulated development environment*
+ 
+    1. [**Create a virtual environment**](#1-create-a-virtual-environment) to avoid collision with our OS and other projects
+    2. [**Fork the pyro-risks project**](#2-fork-the-repository) to be able to start working on a local copy of the project
+    3. [**Set origin and upstream remotes**](#3-set-origin-and-upstream-remotes-repositories) repositories: 
+    4. [**Install project dependencies**](#4-install-project-dependencies)
+   
+* [Contributing workflow](#contributing-workflow): *pull remote changes/new contributions and push your contributions to the original project*
+
+* [Follow the repository style conventions](#style-conventions)
+
+### Project Setup
```

Comment:
Do we really need something as detailed as that ?

Reply:
No we don't, but this description is not for us but for making contributing to this project as easy as possible for anyone that is interested by pyronear regardless of their experience with git, github and open source workflows. 

If it can help just one contributor then it's worth it.